en/flight_modes_rover/manual.md

```diff
@@ -0,0 +1,88 @@
+To manually move/drive the vehicle you move the sticks outside of the centre.
+
+- [**Differential Rover**](../frames_rover/differential_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _yaw rate_ of the vehicle.
+
+- [**Ackermann Rover**](../frames_rover/ackermann_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _steering angle_ of the vehicle.
+
+::: info
+The rover does not attempt to maintain a specific orientation or compensate for external factors like slopes or uneven terrain!
+The user is responsible for making the necessary adjustments to the stick inputs to keep the rover on the desired course.
+:::
+
+## Acro Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Acro mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate estimate.
+:::
+
+Acro Mode is similar to [Manual mode](#manual-mode), but with closed-loop yaw rate control.
+In this mode, the left-stick input remains open-loop for forward speed control, while the right-stick input commands a desired yaw rate setpoint, which is then maintained by the rover's closed-loop control system.
+
+See the tuning for [Acro mode](../frames_rover/differential_rover.md#acro-mode) section to go through the necessary setup to use Acro mode for differential rovers.
+
+## Stabilized Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Stabilized mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate and yaw estimate.
+:::
+
+As with [Acro mode](#acro-mode), the left-stick input remains open-loop for forward speed control, while the right stick input commands a desired yaw rate setpoint, which is then maintained by the rover's closed-loop control system.
+
+In addition, the rover will do closed loop yaw (or heading) control when the right stick is centered, or in other words if the yaw rate setpoint is zero.
+This means that the rover will maintain the current heading if the user only gives throttle input, but no yaw rate input.
+Compared to [Acro mode](#acro-mode), this mode is much better at driving in a straight line as it can more effectively reject disturbances.
+
+See the tuning for [Stabilized mode](../frames_rover/differential_rover.md#stabilized-mode) section to go through the necessary setup to use Acro mode for differential rovers.
+
+## Position Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Stabilized mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate, yaw, speed and position estimate.
+:::
+
+This is the most advanced manual mode. It uses _closed loop yaw rate control_, _closed loop speed control_ and a special logic for driving a straight line when there is no yaw rate input.
```

Comment:
"This is the most advanced manual mode. " - true but what do you mean by _advanced_? Or to put it another way, in MC we order from easiest/safest/most controlled to hardest/least controlled. The point being that for users we're assuming they want to start safe and work up to hard so we introduce the safest mode first.

This also affects the dummy version explanation. Who is this mode for?

Reply:
Reworded this [here](https://github.com/PX4/PX4-user_guide/pull/3377/files#diff-dd65fbcd095c40befe925ee616cdb0679de431aac441983568baf7b8e23ed9c5R77-R93).

Reply:
Yes, like this a lot.